app/views/users/index.html.erb

```diff
@@ -1 +1,3 @@
-<h1>This is a list of users<h1>
\ No newline at end of file
+<% @users.each do |u|%>
+<%= render "single_user", u: u %>
+<% end %>
```

Comment:
[OPTIONAL]

- It is not a good practice to write logic or loop in views. I have shown you below what is the better practice instead of using `@objects.each`.

```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```